src/NuGet.Clients/NuGet.CommandLine/NuGetResources.resx

```diff
@@ -5433,4 +5433,7 @@ Please consider migrating '{0}' to `PackageReference` and using the pack targets
 You can set the '{1}' environment variable to 'true' to temporarily reenable this functionality.</value>
     <comment>Please do not localize `project.json` pack and `PackageReference`. 0 - path, 1 - env var name</comment>
   </data>
+  <data name="Error_NuGetExeNeedsToBeUnblockedAfterDownloading" xml:space="preserve">
+    <value>NuGet.exe file on path {0} needs to be unblocked after downloading.</value>
```

Comment:
Team Review: Let's come up with a better error message is easy to understand and/or actionable

Reply:
`NuGet.exe file on path {0} needs to be unblocked after downloading using Powershell or right-clicking the file and manually unblocking it.`

Maybe?

Reply:
Unblock-File -Path .\nuget.exe

> `NuGet.exe file on path {0} needs to be unblocked after downloading using Powershell or right-clicking the file and manually unblocking it.`
> 
> Maybe?

`NuGet.exe file on path {0} needs to be unblocked after downloading using Powershell command (Unblock-File -Path {0}) or select properties by right-clicking the file in Windows Explorer, then check Unblock next to Security.`
 
How about this one?

Reply:
A good practice is to avoid very specific instructions, but rather state the problem and refer to a something that'd guide them to a solution. 

https://docs.microsoft.com/en-us/windows/win32/debug/error-message-guidelines#best-practices

In particular 

> Provide the user with a solution to the problem. **If the solution has more than one step**, refer to a help topic the explains the task in detail.

Reply:
I'm also not confident the whole downloaded through powershell matters, so I wouldn't even include it at all.
There are many ways to get into that scenario and we shouldn't assume what we don't know.

Reply:
One problem is there is no MS guide on this issue at all on this issue, but there're many 3rd party ones.  
How about we just refer NU5133 error doc https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5133 here? Inside it I can include illustrated guide, picture speaks better than words. Of course, it doesn't exist now, but I can do it before it releases to public. 
Another solution is create NuGet help topic on it first(not NU5133), then put link here.

Reply:
How about I create small section in https://docs.microsoft.com/en-us/nuget/consume-packages/install-use-packages-nuget-cli then refer it? Is it possible?

Reply:
@jeffkl 
Based on current discussion may I resolve this comment? We just keep as it's but focus on https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5133 going to have all the instructions for how to resolve.

Reply:
I have issue for documenting NU5133. https://github.com/NuGet/docs.microsoft.com-nuget/issues/2670